index.js

```diff
@@ -8,7 +8,10 @@ module.exports = function () {
         "stan": "log",
         "sus": "warn",
         "cringe": "error",
-        "L": "Error"
+        "L": "Error",
+        "itsGiving": "return",
```

Comment:
```suggestion
        "spit": "return",
```

Reply:
I quite liked itsGiving haha. Could you keep both?